app/frontend/containers/FundraiserView/FundraiserView.js

```diff
@@ -18,45 +18,47 @@ import {
 
 import type { Dispatch } from 'redux';
 import type { AppState } from '../../state';
+import type { Member, Fundraiser, Page } from '../../state';
 
 type OwnProps = {
-  currentStep: number;
-  formValues: any;
-  member: any;
-  currency: string;
-  donationBands: {[id:string]: number[]};
-  donationFeaturedAmount: ?number;
-  donationAmount: ?number;
-  changeStep: (step: number) => void;
-  selectAmount: (amount: ?number) => void;
-  selectCurrency: (currency: string) => void;
-  setSubmitting: (submitting: boolean) => void;
-  submitting: boolean;
-  fields: any[];
-  outstandingFields: string[];
-  formId: number;
-};
+  fundraiser: Fundraiser;
+  member: Member;
+  page: Page;
+  changeStep: (number) => any;
+  selectAmount: (?number) => any;
+  selectCurrency: (string) => any;
+  setSubmitting: (boolean) => any;
+}
 
 export class FundraiserView extends Component {
-  props: OwnProps & mapStateToProps;
+  props: OwnProps;
 
   componentDidMount() {
-    if (this.props.fundraiser && this.props.fundraiser.amount > 0) {
-      this.props.selectAmount(this.props.fundraiser.amount);
+    const { donationAmount } = this.props.fundraiser;
+
+    if (donationAmount && donationAmount > 0) {
```

Comment:
That's mainly to satisfy flow's type checker which discourages comparing undefined / void to numbers.